src/Private/Helpers.ps1

```diff
@@ -3059,3 +3075,235 @@ function Test-PodeVersionPwshEOL {
+                $Type = 'HashTable'
+            }
+            elseif ($Type -ieq 'List`1') {
+                $Type = 'Array'
+            }
+            elseif ($InputObject -is [array]) {
+                $Type = 'Array'
+            } # whatever it thinks it is called
+            elseif ($InputObject -is [hashtable]) {
+                $Type = 'HashTable'
+            } # for our purposes it is a hashtable
+
+            $output += switch ($Type.ToLower()) {
+                'string' {
+                    $String = "$InputObject"
+                    if (($string -match '[\r\n]' -or $string.Length -gt 80) -and ($string -notlike 'http*')) {
+                        $folded = [System.Text.StringBuilder]::new(">`n") # signal that we are going to use the readable 'newlines-folded' format
+                        foreach ($item in $string.Split("`n")) {
+                            $workingString = $item -replace '\r$'
+                            $length = $item.Length
+                            $IndexIntoString = 0
+                            $wrap = 80
+                            while ($length -gt $IndexIntoString + $Wrap) {
+                                $BreakPoint = $wrap
+                                $earliest = $workingString.Substring($IndexIntoString, $wrap).LastIndexOf(' ')
+                                $latest = $workingString.Substring($IndexIntoString + $wrap).IndexOf(' ')
+                                if (($earliest -eq -1) -or ($latest -eq -1)) {
+                                    $BreakPoint = $wrap
+                                }
+                                elseif ($wrap - $earliest -lt ($latest)) {
+                                    $BreakPoint = $earliest
+                                }
+                                else {
+                                    $BreakPoint = $wrap + $latest
+                                }
+                                if (($wrap - $earliest) + $latest -gt 30) {
+                                    $BreakPoint = $wrap # in case it is a string without spaces
+                                }
+
+                                $null = $folded.Append( $padding).AppendLine( $workingString.Substring($IndexIntoString, $BreakPoint).Trim())
+                                $IndexIntoString += $BreakPoint
+                            }
+                            if ($IndexIntoString -lt $length) {
+                                $null = $folded.Append( $padding).AppendLine( $workingString.Substring($IndexIntoString).Trim())
+                            }
+                            else {
+                                $null = $folded.AppendLine()
+                            }
+                        }
+                        $folded.ToString()
+                        break
+                    }
+                    else {
+                        if ($string.StartsWith('#')) {
```

Comment:
The check for starting with a `#` needs extending to other characters as well, like `[` and `]`:
```powershell
if ($string -match '^[#\[\]@\{\}\!\*]') {
```

Otherwise the editor show indentation failures on some regex lines, and other string values:
![image](https://github.com/Badgerati/Pode/assets/1483395/fcc782e7-6912-46d6-b0e8-95135020c5fd)